sbin/build.sh

```diff
@@ -204,6 +130,7 @@ stepIntoTheWorkingDirectory()
 
 runTheOpenJDKConfigureCommandAndUseThePrebuildConfigParams()
 {
+  cd "$OPEN_JDK_DIR" || exit
```

Comment:
please change to `"${OPEN_JDK_DIR}"`, or perhaps would `"${OPENJDK_DIR}"` be better here?